cmd/upgrade/accountroles/cmd.go

```diff
@@ -310,56 +311,67 @@ func buildCommands(prefix string, accountID string, isUpgradeNeedForAccountRoleP
 		for file, role := range aws.AccountRoles {
 			name := aws.GetRoleName(prefix, role.Name)
 			policyARN := aws.GetPolicyARN(accountID, fmt.Sprintf("%s-Policy", name), policyPath)
-			policyTags := fmt.Sprintf(
-				"Key=%s,Value=%s",
-				tags.OpenShiftVersion, defaultPolicyVersion,
-			)
-			iamRoleTags := fmt.Sprintf(
-				"Key=%s,Value=%s",
-				tags.OpenShiftVersion, defaultPolicyVersion)
-			tagRole := fmt.Sprintf("aws iam tag-role \\\n"+
-				"\t--tags %s \\\n"+
-				"\t--role-name %s",
-				iamRoleTags, name)
+			policyTags := map[string]string{
+				tags.OpenShiftVersion: defaultPolicyVersion,
+			}
+			iamRoleTags := map[string]string{
+				tags.OpenShiftVersion: defaultPolicyVersion,
+			}
```

Comment:
NP: these ares the same tags, could we use the same variable in both cases?

Reply:
Yeah I noticed that, not sure why it was like that already. But we can